uma/uma.go

```diff
@@ -1011,3 +1020,64 @@ func VerifyPostTransactionCallbackSignature(
 	}
 	return verifySignature(*signablePayload, *callback.Signature, otherVaspPubKeyResponse)
 }
+
+func CreateUmaInvoice(
+	receiverUma string,
+	amount uint64,
+	receivingCurrency protocol.InvoiceCurrency,
+	expiration uint64,
+	callback string,
+	isSubjectToTravelRule bool,
+	requiredPayerData *protocol.CounterPartyDataOptions,
+	commentCharsAllowed *int,
+	receiverKycStatus *protocol.KycStatus,
+	invoiceLimit *uint64,
+	senderUma *string,
+	signingPrivateKey []byte,
+) (string, error) {
```

Comment:
Should we be returning a string here or the invoice object directly?

Reply:
Changed to object